### ISSUE TITLE
Issue: getrawtransaction coinbase genesist block

### ISSUE THREAD

**Issue by mhitcher** [gdassori/spruned#48]
```
$ ./bitcoin-cli getrawtransaction 4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b
error code: -32000
error message:
Server error
```
using other transactions work as espected #

**Comment by gdassori** [comment]
Even with the original bitcoind you can't get the genesis block transaction. So don't expect to do so :-)

However, the error message needs to be conformed as the one in bitcoind, so I track this bug.

**Comment by gdassori** [comment]
wip in PR #50

**Comment by gdassori** [comment]
fixed in d0420f69207ad24aad409d82d49906de3fdf1211
